app/views/va_facilities/_facility_status_definitions_modal.html.erb

```diff
@@ -1,7 +1,12 @@
-<a href="#facility-status-modal" aria-controls="facility-status-modal" data-open-modal class="fas fa-question-circle cursor-pointer">
+<%
+  require 'securerandom'
+  guid = SecureRandom.hex(10) #or whatever value you want instead of 10
+%>
+<div href="#facility-status-modal-<%= guid %>" class="cursor-pointer <% if is_map_filter != true  %> margin-bottom-2 modal-status-def <% end %>" aria-controls="facility-status-modal-<%= guid %>" data-open-modal>
```

Comment:
make sure we use `<a>` here too since `<div>`s don't have `href` attributes

Reply:
Got the tags converted to <a tags for the links to open the modals.